### PR TITLE
Incident details

### LINKED ISSUE #9: GET Incident's Details

### PR DESCRIPTION
**Pull request by amsmyth1**
### What Changed?
- Create endpoint to retrieve an Incidents Details
- Create error handling for sad and edge cases
### What's Next?
- Update an incident
### Known Issues?
- none
### Test Coverage?
- 100%
### README & Documentation
- Update documentation in README for json response to incident details
### Closed Issues
Close #9

### CODE REVIEW

File: spec/requests/api/v1/get_incident_details_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      result = JSON.parse(response.body, symbolize_names: true)
+
+      expect(response.status).to eq(404)
+      expect(result).to be_a(Hash)
+      expect(result[:error]).to eq("Incident does not exist with that id")
+    end
+    it "edge case" do
+      incident = create(:incident, active: true)
+
+      get "/api/v1/incidents/ashjdnvajhnv"
```
Comment (klatour324) [inline spec/requests/api/v1/get_incident_details_spec.rb:35]: This is great edge case testing! Another thing Tim mentioned to me last mod that we can test for as well would be 'odd characters' as an edge case. For example if it's a bunch of &&&& and !%&$%* or [].

**Review by klatour324 — APPROVED** [review]
This all looks great Alexa! Left a note for future considerations based on feedback I received last mod that can be helpful for all of us when we are looking for edge case testing. Otherwise, this is looking great!

**Review by jmichlitsch — APPROVED** [review]
I like the edge case testing for id and the errors messages you are returning in the controller.
